ccswbs/gus
# Remove tags from Program page and replace with description

# Summary of changes
In the program page template, remove tags from the area below the main heading and replace them with the program's Open Graph metatag description.

## Frontend
- remove `Tags` component call from `program-page.js` and replace it with `ogDescription`
- limit Call to Action in same area to one button only
- refactor `callToAction.js` for Bootstrap 5
- remove custom stylesheet from `program-page.js`
- refactor and simplify various conditional statements and variable declarations
- replace `contentIsNullOrEmpty` function calls with `contentExists`
- remove unneeded functions from `ug-utils.js`
- refactor `employers.js` for Bootstrap 5
- add `no-icon` class to `global.css` to remove unwanted icons from list items
- add `two-col-md` class to lists in `variants.js`

## Backend
- None

# Test Plan

- Go to https://tqtest.gatsbyjs.io/programs/
- View https://tqtest.gatsbyjs.io/programs/associate-diploma-in-turfgrass-management and ensure the former Tag area does not appear (this program has no description or CtAs)
- View https://tqtest.gatsbyjs.io/programs/bachelor-of-arts-general-fully-online and ensure the description appears in the former Tag area along with the first Call to Action button only
- Ensure information in tabbed areas appears correctly 
- Review other programs to ensure no formatting is amiss

# Known Issues
The BAG program is the only one with a proper Open Graph metatag description. Until this value is filled in for other programs, they will display only the Call to Action button in the dark grey area beneath the program title.
